packages/jest-runner/src/runTest.ts

```diff
@@ -24,22 +23,26 @@ import {
   setGlobal,
 } from 'jest-util';
 import LeakDetector from 'jest-leak-detector';
+import Resolver from 'jest-resolve';
 import {getTestEnvironment} from 'jest-config';
 import * as docblock from 'jest-docblock';
 import {formatExecError} from 'jest-message-util';
 import sourcemapSupport from 'source-map-support';
 import chalk from 'chalk';
+import {TestFramework, TestRunnerContext} from './types';
 
 type RunTestInternalResult = {
-  leakDetector: ?LeakDetector,
-  result: TestResult,
+  leakDetector: LeakDetector | null;
+  result: TestResult.TestResult;
 };
 
 function freezeConsole(
+  // TODO: clarify this in the PR
+  // why cant the names be found even though I've exported them
   testConsole: BufferedConsole | Console | NullConsole,
-  config: ProjectConfig,
+  config: Config.ProjectConfig,
 ) {
-  testConsole._log = function fakeConsolePush(_type, message) {
+  testConsole.log = function fakeConsolePush(_type: unknown, message: string) {
```

Comment:
testConsole._log doesnt exist on the type

Reply:
yeah, it's `private`... For now, just add a `@ts-ignore` - we override the private one on purpose. Will need to revisit that